torch/csrc/Module.cpp

```diff
@@ -126,8 +126,14 @@ static PyObject * THPModule_crashIfCsrcASAN(PyObject *module, PyObject *arg) {
   return PyLong_FromLong(x[0]);
 }
 
+static PyObject * THPModule_crashIfCsrcUBSAN(PyObject *module, PyObject *arg) {
+  int32_t x = 0.0;
+  int32_t y = 1.0 / x;
```

Comment:
This spelling makes me nervous, because a compiler that is smart enough can optimize this away (since divide by zero is UB, so anything goes). A better way is to read in an int from arg and divide by it.